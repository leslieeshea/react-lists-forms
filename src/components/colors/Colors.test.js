import React from 'react';
import { shallow } from 'enzyme';
import Colors from './Colors';

describe('Colors component', () => {
  it('renders Colors', () => {
    const colors = [
      { name: 'Red', hex: '#FF0000' },
      { name: 'Blue', hex: '#0000FF' }
    ];

    const wrapper = shallow(<Colors colors={colors} />);
    expect(wrapper).toMatchSnapshot();
  });
});
