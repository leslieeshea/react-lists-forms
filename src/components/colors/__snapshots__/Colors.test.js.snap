// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors component renders Colors 1`] = `
<ul>
  <li
    key="Red"
  >
    <Color
      hex="#FF0000"
      name="Red"
    />
  </li>
  <li
    key="Blue"
  >
    <Color
      hex="#0000FF"
      name="Blue"
    />
  </li>
</ul>
`;
